app/views/steps/dwp/cannot_check_dwp_status/edit.html.erb

```diff
@@ -10,9 +10,11 @@
         <h1 class="govuk-heading-xl"><%= legend_t(:cannot_check_dwp_status_heading)%></h1>
         <p class="govuk-body"><%= legend_t(:cannot_check_dwp_status_subheading)%></p>
 
+      <%= step_form @form_object do %>
         <p class="govuk-body">
-          <%= t('helpers.hint.steps_dwp_cannot_check_dwp_status_form.next_steps_label') %>
+          <%= t('helpers.hint.steps_dwp_cannot_check_dwp_status_form.next_steps_label_html') %>
```

Comment:
Doesn't seem like there were tests previously for this link but it is essentially a page refresh so does it need testing? 🤔